crates/sncast/tests/e2e/deploy.rs

```diff
@@ -286,25 +288,27 @@ async fn test_happy_case_with_constructor() {
 
 #[tokio::test]
 async fn test_happy_case_with_constructor_cairo_expression_calldata() {
+    let tempdir = create_and_deploy_oz_account().await;
+
     let args = vec![
         "--accounts-file",
-        ACCOUNT_FILE_PATH,
+        "accounts.json",
```

Comment:
Can you explain this update?

Reply:
See above, the `let tempdir = create_and_deploy_oz_account().await;`. It deploys a new account and stores it files in the structure that is hardcoded within tempdir.

We change the current_dir of snapbox to tempdir below and there it finds this files.

It's hardcoded in other tests that use `create_and_deploy_oz_account`, don't think that's problematic.